src/net_processing.cpp

```diff
@@ -1788,7 +1788,8 @@ bool static ProcessMessage(CNode* pfrom, const std::string& strCommand, CDataStr
 
         std::list<CTransactionRef> lRemovedTxn;
 
-        if (!AlreadyHave(inv) && AcceptToMemoryPool(mempool, state, ptx, true, &fMissingInputs, &lRemovedTxn)) {
+        if (!AlreadyHave(inv) &&
+            AcceptToMemoryPool(mempool, state, ptx, &fMissingInputs, &lRemovedTxn, false /* bypass_limits */, 0 /* nAbsurdFee */)) {
```

Comment:
Nit, feeling uncomfortable with these comments 😄.

Reply:
I think he's just joking about the existence of "bypass limits" and "absurd fee" options.

Reply:
No joking, just what @sipa said. One struct with the arguments and fill them, resembles named arguments.